### PR TITLE
Gal 296/invitations screen clinic information

### PR DESCRIPTION
**Pull request by msandhuNHS**
## Description
https://nhsd-jira.digital.nhs.uk/browse/GAL-296

As we do not have a custom domain, the API id for the fetch calls will change on each build of the backend thus must be updated accordingly. 

Clinic Information screens implemented.

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Review by joad432 — APPROVED** [review]
Good good!!

**Review by LaithM96 — CHANGES_REQUESTED** [review]
[Invalid workflow file: .github/workflows/deploy-to-s3.yaml#L17](https://github.com/NHSDigital/Galleri-Frontend/actions/runs/6161665835/workflow)
You have an error in your yaml syntax on line 17

**Review by LaithM96 — APPROVED** [review]
I spoke to Manny and this file is not been used just yet, and it shouldn’t have any problem with it.

File: package.json [hunk 1]
```diff
@@ -30,8 +31,7 @@
     "gulp-sass": "^5.1.0",
     "jest": "^29.6.3",
     "jest-environment-jsdom": "^29.6.3",
-    "sass": "^1.64.2",
-    "cypress": "^12.17.4",
```
Comment (joad432) [inline package.json:34]: This being removed, will this affect myself and Laith when it comes to testing? Will we have to install Cypress again?

**Review by joad432 — CHANGES_REQUESTED** [review]
pleasee check comment on package.json

**Review by joad432 — DISMISSED** [review]
Looks good

**Review by zainm97 — APPROVED** [review]

**Review by joad432 — APPROVED** [review]

**Pull request merged**